velocity/src/main/java/net/donationstore/velocity/DonationStorePlugin.java

```diff
@@ -0,0 +1,60 @@
+package net.donationstore.velocity;
+
+import com.google.inject.Inject;
+import com.moandjiezana.toml.Toml;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+import com.velocitypowered.api.event.proxy.ProxyShutdownEvent;
+import com.velocitypowered.api.plugin.Plugin;
+import com.velocitypowered.api.plugin.annotation.DataDirectory;
+import com.velocitypowered.api.proxy.ProxyServer;
+import net.donationstore.exception.ClientException;
+import net.donationstore.logging.Logging;
+import net.donationstore.velocity.commands.DonationStoreCommand;
+import net.donationstore.velocity.config.FileConfiguration;
+import net.donationstore.velocity.logging.Log;
+import net.donationstore.velocity.queue.QueueTask;
+
+import java.nio.file.Path;
+
+@Plugin(id = "donationstore", name = "Donation-Store-Velocity", version = "2.1", authors = {"Donation Store"})
```

Comment:
So this will be the second generation of the plugin, but this is the first version of Velocity so v2.0.